example/src/App.tsx

```diff
@@ -48,6 +48,7 @@ export default function App() {
       <PlatformInfo />
       <MarkdownTextInput
         multiline
+        formatSelection={(selectedText, formatCommand) => formatCommand === 'formatBold' ? `*${selectedText}*` : formatCommand === 'formatItalic' ? `_${selectedText}_` : selectedText}
```

Comment:
> but can you add formatSelection to the example app, please?

Already added it here.

Reply:
Can we wrap this function with `useCallback` or even move it to top-level scope (as `function formatSelection(...) { ... }`)?